src/Components/Home.js

```diff
@@ -35,14 +84,13 @@ function HomePage() {
 
   return (
     <div>
-      <div className="clubTitleSection">
-        <img className="uw-logo" src={UWlogo} alt="UW logo" />
-        <h1 className="clubName">Software Engineering Career Club</h1>
-      </div>
-      <div className="whatWeDoSection">
-        <h2 className="summaryTitle">Who we are</h2>
-        <p className="summary">
-          The Software Engineering Career Club (SWECC) is a student-led organization at the University of Washington in Seattle. Our mission is to support and empower individuals interested in pursuing a successful career in software engineering. We offer a wide range of activities and resources,
+        <Carousel/>
+
+        <div className="whatWeDoSection">
+            <h2 className="summaryTitle">Who we are</h2>
+            <p className="summary">
+                The Software Engineering Career Club (SWECC) is a student-led organization at the University of
```

Comment:
please install the [Prettier code formatter extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) to make the code more neat. If you are using vs code, you can install it from the extensions tab. If you are using Webstorm, this extension is already built in. For any other IDEs, just make sure the code formatter is using Prettier